projects/storefrontlib/src/cms-components/product/product-list/product-facet-navigation/facet.model.ts

```diff
@@ -35,3 +35,7 @@ export interface FacetCollapseState {
    */
   maxVisible?: number;
 }
+export enum ToggleState {
```

Comment:
P2: Why we've introduced this enum? Do we plan adding more values here? If not, I think that boolean should suffice.

Btw. To me, _toggling_ sounds like switching between any two states A/B (i.e. on/off, not only expanded/collapsed). Here we add to the global lib's namespace an enum `ToggleState` with values Expanded/Collapsed, which is a bit too specific IMO. I hope using `boolean` suffices, but maybe I don't see a bigger picture...

Reply:
yeah toggle state is way to generic. I'll rename. 

A boolean is unfortunately not enough. We had the following bug:
- the initial state is driven by CSS (meanin on desktop the first 3 facets are expanded by default)
- after the user "toggled" a facet group, it would keep the toggled state across different pages. However, if the same facet was on a different position on another page, the toggled state was causing an expanded state. 

This is why we now have explicit expanded and collapsed classes.